rocketpy/rocket/rocket.py

```diff
@@ -433,6 +436,26 @@ def evaluate_dry_mass(self):
 
         return self.dry_mass
 
+    def evaluate_structural_mass_ratio(self):
+        """Calculates and returns the rocket's structural mass ratio.
+        It is defined as the ratio between of the dry mass
+        (Motor + Rocket) and the initial total mass
+        (Motor + Propellant + Rocket).
+
+        Returns
+        -------
+        self.structural_mass_ratio: float
+            Initial structural mass ratio dry mass (Rocket + Motor) (kg)
+            divided by total mass (Rocket + Motor + Propellant) (kg).
+        """
+        # Make sure there is a motor associated with the rocket
```

Comment:
what about this comment?

Are you leaving it as a TODO?
```suggestion
        # TODO: Make sure there is a motor associated with the rocket
```

Or should we simply remove it?
```suggestion
```

Reply:
Probably just remove it, I left it as a mistake.